### DISCUSSION TITLE
Welcome to tls-scan Discussions!

### DISCUSSION THREAD

**Discussion by prbinu** [prbinu/tls-scan#39] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ImJeremy0** [comment]
Good eve.. Mr Author Does it have full installation guide for Android phone users?
And the Bulk scanning command?

I am asking for the Script just like this..

Ssl scan installation: 👇

termux-setup-stotage
pkg update && pkg upgrade
pkg install python
pkg install git
git clone github.com/dinotools/sslscan
pkg install sslscan
sslscan

Step by Step Termux Command: 👇
Method#1: 👇

1. nmap -sL -n 104.16.96.0/24 | awk '/Nmap scan report for /{print $NF}' > Bughost1.txt

2. sslscan --no-check-certificate --no-ciphersuites --no-fallback --no-renegotiation --no-compression --no-heartbleed --no-check-certificate --no-sigs --no-heartbleed --no-groups --tlsall --timeout=1s --send-timeout=0s --targets=Bughost1.txt | awk '/Connected/ || /enabled/{print}' > Bughost1res.txt

3. cat result.txt

Could you please give me guide Just like this installation guide and command in sslacan? I just didn't understand, it would be really helpful..🙏🙏

**Reply by prbinu** [reply]
Hi, currently I've tested only on x86 systems.
The build instructions are documented here: https://github.com/prbinu/tls-scan#build-from-source
Built-in binaries are available here: https://github.com/prbinu/tls-scan#pre-built-binary
The usage is documented here: https://github.com/prbinu/tls-scan#example

Hope this helps!
-binu
